metadata/base/metadata-db-configmap.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: db-configmap
+  name: metadata-configmap
```

Comment:
all `name` will be prefixed with `metadata-` by kustomize. So it is better to use `db-configmap` or `mysql-configmap`.